Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM golang:alpine AS build
 
-RUN apk update && apk --no-cache add bash ca-certificates && apk add --no-cache gcc musl-dev
```

Comment:
i dont think the dockerfile was actually broking, as it appeared to work okay in my testing...but i question everything now